linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] [REQUIRED] I can see you have added the `linters.yml` file, but you also need to add the `CSS` and `html` linter files that are in [this page](https://github.com/microverseinc/linters-config/tree/master/html-css). Please ensure to add the [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) and [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) files as these will help analyze your source code to flag programming errors, bugs, stylistic errors, and suspicious construct